src/es/components/controllers/withFacet/WithFacet.js

```diff
@@ -84,11 +84,18 @@ export default class WithFacet extends WebWorker() {
       } else if (event?.type === 'reset-filter') {
         // reset particular filter, ks-a-button
         const filterKey = event.detail.this.getAttribute('filter-key')
-        this.deleteParamFromUrl(filterKey)
         const result = await this.webWorker(WithFacet.updateFilters, currentCompleteFilterObj, filterKey, undefined, true)
         currentCompleteFilterObj = result[0]
         currentRequestObj.filter = result[1]
         if (filterKey === 'q') delete currentRequestObj.searchText
+        if (filterKey === 'cname') {
+          this.deleteParamFromUrl('clong')
+          this.deleteParamFromUrl('clat')
+          delete currentRequestObj.cname
+          delete currentRequestObj.clong
+          delete currentRequestObj.clat
+        }
```

Comment:
The filterKey "cname" has been replaced with "cdist" to comply with the API. Please rename it.

Reply:
So, the filterKey has changed but the Obj.propertyName stays as 'cname'? 
i mean currentRequestObj.cname

Reply:
Command back. It stays with "cname". The API has introduced "cdist", but as long as we don't specify the location name, we don't need to match the API here. Especially since it is not clear from the documentation what cdist is actually used for.